.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+repos:
```

Comment:
Is it worth updating the contributing docs with details of these formatting/linting requirements? https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/blob/main/CONTRIBUTING.md#-submitting-a-pull-request-pr

Reply:
Added!